warehouse/ingest-core/src/main/java/datawave/ingest/config/TableConfigCache.java

```diff
@@ -22,12 +23,13 @@
     protected static final String DELIMITER = "\t";
     protected static Map<String,String> configMap;
     
-    protected static Logger log = Logger.getLogger("datawave.ingest");
+    protected static final Logger log = Logger.getLogger("datawave.ingest");
     
     public TableConfigCache(Configuration conf) {
         super(conf);
     }
     
+    @SuppressFBWarnings("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD")
```

Comment:
It seems like this was the best action to take here - it was complaining that we were writing to a static field from a non-static method. Perhaps there's another way to do this that would not result in a warning?

Reply:
I'll investigate this when I get back. Approving for now. Long term we shouldn't  need this suppression.

Reply:
Looks like `configMap` is static but we're setting it in a non-static method. The method actually has no dependency on any instance state. Seems like the fix might be to make this method static.

Reply:
@jschmidt10 - I suspected the same thing but was going to leave the design up to @hlgp.

Reply:
Sounds good!